db/migrations/state/0018.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+ALTER TABLE state.block
+    ADD COLUMN IF NOT EXISTS checked BOOL NOT NULL DEFAULT TRUE;
```

Comment:
In a full resync it is perfect but in the current env, If we do so, maybe we miss some previous mismatch. becuase we were assuming tha it was ok. Wdyt?